projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -225,7 +225,7 @@ control_service_publish_api_client:
 # Go to the namespace -> StorageClass
 control_service_deploy_testing_data_pipelines:
   stage: pre_release
-  image: proum/aws-iam-authenticator
+  image: intellibank/aws-iam-authenticator:0.5.12
```

Comment:
Yes, I think you need some testing . You can run it with docker run and see if the script would work at least teh AWS authentication part - https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/cicd/deploy-testing-pipelines-service.sh ( which really is KUBECONFIG and kuberenetes commands really). 

Just now running it locally , itseems there's no shell installed - `docker run --entrypoint sh --network="host" -v /Users/aivanov/dev/workspace/github:/repo -it intellibank/aws-iam-authenticator:0.5.12` 

While the previous did have "sh" installed. Without shell the below script seems unlikely to be able to run.

Reply:
I don't think it's big deal to install awi-iam-authenticator ourself . It's one line curl command - https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html